packages/cozy-doctypes/src/banking/BankAccountStats.js

```diff
@@ -13,6 +13,8 @@ class BankAccountStats extends Document {
       return sums
     }, {})
 
+    summedStats.currency = 'EUR'
```

Comment:
Don't you have the currency in your source data ? It bugs me to have this hardcoded there

Reply:
Vu ensemble. Le EUR est dans les données sources. Si les données ne sont pas toutes sur la même devise de toutes façons la somme n'a aucun sens donc il faudra stopper en amont et ne pas créer le doc de stat (qui ne voudrait rien dire)